### PR TITLE
Tn ra migrate 60 712

### PR DESCRIPTION
**Pull request by JohannaOtt**
SVC-1474

### CODE REVIEW

**Review by florianesser — COMMENTED** [review]
@JohannaOtt Do the mapping table line numbers in the commit messages refer to the Excel sheet line numbers or the numbers in column A? I have the feeling two of the commits in this PR handle that differently.

**Comment by JohannaOtt** [comment]
> Do the mapping table line numbers in the commit messages refer to the Excel sheet line numbers or the numbers in column A?

They refer to the Excel sheet line numbers and from what I just checked, it looks like that is consequently used:
* Line 3846: new value "Personenverkehr" for AX_Bahnverkehr.bahnkategorie
* Line 3869: new value "Personenverkehr" für AX_Bahnstrecke.bahnkategorie

File: annex-1/mappings7.1/RailroadTransportNetwork/aaa-tn-ra-01.halex.alignment.xml [hunk 1]
```diff
@@ -193,7 +193,9 @@ def railwayTypes = []
 _source.p.bahnkategorie.each { bkt -&gt;
 	switch (bkt) {
 	case '1100': // Eisenbahn
+	case '1101': // Personenverkehr
```
[the remark this answers is not in the document]
Comment (JohannaOtt) [reply annex-1/mappings7.1/RailroadTransportNetwork/aaa-tn-ra-01.halex.alignment.xml:196]: No, this one is part of the AX_Bahnverkehr changes whereas the other one is part of the AX_Bahnstrecke changes.

[the remark this answers is not in the document]
Comment (florianesser) [reply annex-1/mappings7.1/RailroadTransportNetwork/aaa-tn-ra-01.halex.alignment.xml:196]: Ah, thanks for the clarification

**Review by florianesser — APPROVED** [review]
Nitpick on the last commit message: using a Markdown link is slightly better IMO

```
feat: add handling for new value of AX_Bahnverkehr.zustand

Added mapping based on customer feedback in [this hale»connect task](https://haleconnect.com/#/transformation/org/765/f80c4a8a-0553-45a8-9e3e-efa75ba5f79b/tasks/o55j).

SVC-1474
```

Otherwise LGTM

**Pull request merged**